### ISSUE TITLE
[Bug]: Emojis in filenames result in a "400 Bad Request" error

### ISSUE THREAD

**Issue by moritzlang** [remotely-save/remotely-save#93]
### What happened?

Emojis in filenames result in a "400 Bad Request" error

### What OS are you using?

iOS (iPhone, iPad)

### What remote cloud services are you using?

webdav

### Version of the plugin

0.3.19

### Version of Obsidian

1.2.0

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
remotely save doesn’t restrict any emoji in file names. would you like to try directly create an emoji file on webdav server (by webdav webpage or other client, not remotely save), to check whether the server supports emoji files?

**Comment by fyears** [comment]
I tried to sync the file to a webdav server, and the files names with emoji can be synced correctly. So maybe you could adjust some settings of the server to make it support emoji file names?

Another workaround is using an end-to-end encryption password, so that the file names are encrypted and will not contain emoji before being sent to the server.

**Comment by moritzlang** [comment]
@fyears You are right, this was indeed a limitation of my nextcloud installation. Thanks for clearing that up!
